packages/cos/build.yaml

```diff
@@ -19,15 +19,10 @@ requires:
 - name: "grub-config"
   category: "system"
   version: ">=0"
-
-# Skip selinux policies for recovery
-{{if ne .Values.category "recovery" }}
```

Comment:
Any specific reason for re-adding it? In case it's needed, we also need to update the recovery bootargs

Reply:
Yes, because then at deploy time the labels can't be applied, it requires SE Linux binaries to apply the labels.